File: chat-service/src/main/java/com/yazikochesalna/chatservice/config/properties/UserServiceProperties.java
package com.yazikochesalna.chatservice.config.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Component
@ConfigurationProperties(prefix = "user.service")
@Data
public class UserServiceProperties {
    private String url;
}

File: authorization-service/src/main/kotlin/com/yazikochesalna/authorizationservice/config/properties/UserServiceProperties.kt
package com.yazikochesalna.authorizationservice.com.yazikochesalna.authorizationservice.config.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component


@Component
@ConfigurationProperties(prefix = "user.service")
class UserServiceProperties {
    lateinit var url: String
}
